src/components/applicant-profile/ApplicantProfile.styles.js

```diff
@@ -15,15 +15,20 @@ export const useStyles = makeStyles(theme => ({
     gap: "1px 1px",
     "& h1": {
       fontSize: "2em",
-      marginBlockEnd: "0.5em",
       [theme.breakpoints.down("xs")]: {
         margin: "0"
       }
     },
     "& h2": {
+      fontSize: "1.2em",
       fontWeight: "normal",
-      marginBlockEnd: "0",
-      marginBlockStart: "1.2em"
+      marginBlockStart: "1.8em",
+      [theme.breakpoints.down("xs")]: {
```

Comment:
oh, you have a really good handle on these breakpoints now! I'll be coming to you for tips